charts/portworx/Chart.yaml

```diff
@@ -1,5 +1,5 @@
 name: portworx
-version: 2.9.1
+version: 2.11.0
```

Comment:
Let's update the version based on the px version. Check the other comment.

Reply:
Ok, I will keep it unchanged then, probably dont need the version suffix .4